.env

```diff
@@ -133,3 +133,15 @@ COMMS_TEMPLATE_PORT=8182
 
 # EQ
 EQ_URL=https://eq-test/session?token=
+
+# Respondent Home UI
+RESPONDENT_HOME_UI_PORT=9292
+RESPONDENT_HOME_ANALYTICS_ACCOUNT=UA-56892037-7
+RESPONDENT_HOME_EQ_HOST=eq-server
+RESPONDENT_HOME_EQ_PORT=5000
+RESPONDENT_HOME_EQ_PROTOCOL=http
+RESPONDENT_HOME_IAC_SERVICE_PROTOCOL=http
+RESPONDENT_HOME_LOCALE=en
+# base64 encoded keys to workaround newlines issue with .env files (https://goo.gl/GXqLCv)
+RESPONDENT_HOME_EQ_PRIVATE_KEY=LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBcTByRnViL2ZDMlM0a2NQNGw5aXZ4Z2xqaDhsM2hZT0tHNXo4bUoxY2dnb1JxVGFJCnBlUzNkOGVNNHpMcHVZMFBwNUdNdWxIZERUamZvZ3VQT1NCMk9KaVlVbThRRndpcTJMNEM5MlRzdGVSWGFhdEoKV3FPUnlIMktQOFg3cTA5ZnF0dUlpQlh2RDlrd0E5TWlpZCtaOVEvWWU5NTBHVGFKRUowbm9ycVpwWkVjSVJkbQpsWkllbHFYSnhFOE9WK0hiRW9qY2NMVTdSYUw3YXZDZVZuSzNqaHpRbW9mQW1FRjNyTHFFV0hwY3R5dU8wVGZFCkV3OE9ieHMyK2tsc1FXbnNmcTNMaEI3Z3JtM2RiVHJUV0JPU2NQb2xGa3VJN1RiYWJJRmNPeHkwZ3hKSG1GOVQKMWFQYXltL2cwS1ZPTGhmQUVJNTJEVjdxNGN5d0h2NEZDSzZkQndJREFRQUJBb0lCQUI0L2ZQMFpNZFQ4TzArYQpVS3JYMitwWHZCaTliZnJJeUxtcGxqMkpCRTMrWXpJRGJFd0hRSkxKVFVyTnNkSlBZNlZPb1pCQlpveXdMMldaCkx2aUNqLzNjaklGNkZWaEw4b2VGdmxaYm55VmJwa2dGdEZWcUx4TTlnQTFVOCtWRGhmeFEraUkxNVZwaUd3V0IKNGthQWt0cGdxdlRzeW9tdFJMVUhXM2wwQkMvbFR2ZjJuUXcrVGdqQ3A1REd5MlZaUHJ6RG9uQkl4cHBEMTJEdQoyYTRGWXVrejJQbno3dkNGSHBFZXB6K05XbW1tZlJITXMvTGd4Q0NFay9MZkUzeVNVbU5kYWdDQjc4NEJKeEpNCk51czF2cmd1ckpFOU1XT3QxVFE4bHlCTnRwYUdvNXBUazBWRmN2RndlNWxOcCtmaU1YdmkxbkxPR3FVeXNzTEMKVXhjMURZa0NnWUVBMzluK0ZZTnI4NlhjOENjVlVBUXlyM01LRG5rNVRPVHk2b1VVWHBKcTBFWkRvcWd1cVJmZwo5aU1sWnlqL1FxemNvdDdBaXh3dUJORmdBTGlFWHNwZDNROXVTTHh1d0xLc1dlT3JJc3ZVR1pRd2xkSXRTSUQzCjd1d2I4UXIzVE9BWm1PUGlwYVFsNzFxY0E0STA4bHErU3A5S2ZkMEVORVcyV0FMVFRDMzBmUVVDZ1lFQXcrUm8KSEFkNnJ4Q0l5REloNm5uazY5UFM5K2l1UjZILzZCYzB2VTFhYWdzekRRa3RwazBVRGtqYWFNUTZSSGVialpqVwpzekJyc0Q1RUZUNUdMeExqMHZiVUp2V3pLNis2bHFPemlzZGlJdTBVMTFCM2VMNk5FdmFBZHNnU3E5UWo0WDZOCnJZaDV4MFc1RTJWSW9EQWUxTmlnbEphZ2lhSFV2RjgyUzE1WEw1c0NnWUVBa3hJMFA0b3krTWQybTBueVZMb04KV3Z4b0ZRNU1PVnNLeGs1ZERhMkJCQW5HRWk0UlNnTm9TQTdiV2lXelgra1dwT05ZK1l5d0ozWXdid2pMMzBOeAoyL1Z4YmRSdnA2YzhXV3FkVzZVeld3TGowT3RmMlF6eTJFVEVMdUFYanFpRjZNK0FiM2VqZk5pNVBBem1JSHpDCkhXRCtuaWNCeGg0dktTL2lCMkVDQ3NrQ2dZQUx0SDQwRG81YUg1Yk12NUlmUEtGZEpnOFpHU0l3elpGSkFQbjUKeGNGOVRLWUVSV1FnNThJbkNGdGtuelVNR2ttRk1IakR4RVRyQncxeVo0SXlGZUxmSERpWkMxMWxZdDYzV1pqRQoxaDBiRVNlSnBoazljY1ZrT0s1OGFvQVRIRjM0ZjJOZjF5UFBzRzdhcGRhdEc3TDc3T1hMZXlTa2xMTHRmcEx6CjlmTW51UUtCZ0ZKQ2xoMnZOWUZsTm5kWG9WNU5GUExOejFMNVd2NVdOd1djdUdPNGlDc1VoMmpGVmw0M3o0NTkKVjdadVJwLy80dkl6RU9OcXF5TFVGdDFKWkt1Q3dXUUIyc1JqS1dwcVJCMDdDMG8yNDNYZWZENWY3ODQ0bm1YcQpIMHhLTUZSWWpEcWhGZFRYRldWUG5rL3R1L014b1NUZEswRmZzZ1VuanMrVEd2TjVUbkJXCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCgo=
```

Comment:
Is this the private key unencrypted?

Reply:
Yeah, after it has been base64-decoded it gives the private key data similar to this:
```
-----BEGIN RSA PRIVATE KEY-----
...key...
-----END RSA PRIVATE KEY-----
```
I used keys from this repo (https://github.com/onsdigital/ras-frontstage/blob/fee057c93a17388d8f042a3833c33daf2742cfa7/tests/test_data/jwt-test-keys/test_key.json#L1)